src/api/app/controllers/webui/apidocs_controller.rb

```diff
@@ -18,7 +18,7 @@ def file
       send_file( file, :type => "text/xml", :disposition => "inline" )
     else
       flash[:error] = "File not found: #{params[:filename]}"
-      redirect_back_or_to :controller => 'apidocs', :action => 'index'
+      redirect_back(fallback_location: {:controller => 'apidocs', :action => 'index'})
```

Comment:
I could but it was already like that and it has nothing to do with the fact that `redirect_back` is deprecated. :no_mouth: